libraries/standard/mqtt/cbmc/include/mqtt_config.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MQTT_CONFIG_H_
+#define MQTT_CONFIG_H_
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Configure name and log level for the MQTT library. */
+#define LIBRARY_LOG_NAME     "MQTT"
+#define LIBRARY_LOG_LEVEL    LOG_NONE
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/* Set network context to a socket (int). */
+typedef int NetworkContext_t;
```

Comment:
Where did this come from?  Is this the hack of setting NetworkContext to an int (or int *) because in the real code it is typedef'd to an SSL struct?   I would document what is going on here, an that this is replacing a definition in some header file.

Reply:
It can be typedef to whatever we want. We don't use it in the network interface stub.

Reply:
I really think a comment saying this is a stub would be valuable.

Reply:
ok